internal/core/context.go

```diff
@@ -89,10 +89,7 @@ func GetOrganizationIDFromContext(ctx context.Context) string {
 
 func GetProjectIDFromContext(ctx context.Context) string {
 	client := ExtractClient(ctx)
-	projectID, exists := client.GetDefaultProjectID()
```

Comment:
Hum I'm not really comfortable with silencing this test of existence :/

Reply:
This is the same behavior as `GetOrganizationIDFromContext(ctx context.Context) string`, a couple lines before this one.
Goal is that the Default function should not fail if there is no default project id. As Default function is used when generating documentation

Reply:
The end goal is a fix on the API side, though it's a temporary solution. This function is only used here, and if there is no project ID, the API will throw an error.